llama_stack/distribution/routers/routers.py

```diff
@@ -211,16 +212,16 @@ async def shutdown(self) -> None:
     async def score_batch(
         self,
         dataset_id: str,
-        scoring_functions: List[str],
+        scoring_functions: Dict[str, Optional[ScoringFnParams]] = None,
```

Comment:
uh oh custom router code :) 

don't worry I am just noting this.

Reply:
Yeah, the scoring functions needs to be looped over for the router to look up different providers.